packages/editor-ui/src/styles/plugins/_vueflow.scss

```diff
@@ -100,3 +106,20 @@
 .vue-flow__edge-label.selected {
 	z-index: 1 !important;
 }
+
+/**
+ * Controls
+ */
+
+.vue-flow__controls {
+	margin: var(--spacing-s);
+
+	@include mixins.breakpoint('xs-only') {
+		max-width: calc(100% - 42px - var(--spacing-s));
```

Comment:
42 :D 
![CleanShot 2025-01-06 at 09 39 18](https://github.com/user-attachments/assets/3cb729b2-f49d-4b92-94da-e8963e6624bc)

Reply:
It's magic! Updated to use variables 🙈